src/components/card/CdrCard.vue

```diff
@@ -1,3 +1,17 @@
+<script setup>
+import { useCssModule, defineProps } from 'vue';
```

Comment:
You don't need to import `defineProps` it's a compiler macro that's built in to the syntax.